airbyte-integrations/connectors/source-trello/source_trello/source.py

```diff
@@ -55,6 +55,13 @@ class ChildStreamMixin:
     parent_stream_class: Optional[TrelloStream] = None
 
     def stream_slices(self, sync_mode, **kwargs) -> Iterable[Optional[Mapping[str, any]]]:
+        config = self.config
```

Comment:
it seems we don't need to assign to local `config` variable here 
we can just use `self.config`